### DISCUSSION TITLE
Pawn Spectator Gadget 0: Slot param missing

### DISCUSSION THREAD

**Discussion by erSuyognp** [SimDynamX/SpaceTeamsPRO#42] [2024 Lunar SAR Competition Q&A] [answered]
Hi, I am having this problem where everything loads but nothing moves. I thought it was problem with my code. But even if I try to run a demo that works on tutorial don't work on mine. When sim starts, I always get this error : "Pawn Spectator Gadget 0 : Slot param missing", I tried to follow BP path I can't find any file like Game or Spacecraft_UI (Maybe I am not supposed to see them). But yeah, I can't run my sim. 

![Screenshot 2024-10-27 185149](https://github.com/user-attachments/assets/5f5f1804-1733-41a9-8d9b-c52588745c94)

**Comment by connorjak** [comment, marked as answer]
This is a known error with many of our outdated sims; see my response in 
- https://github.com/SimDynamX/SpaceTeamsPro/issues/43
